SteamKit2/Tests/CallbackManagerFacts.cs

```diff
@@ -26,13 +26,13 @@ public void PostedCallbackTriggersAction()
             var callback = new CallbackForTest { UniqueID = Guid.NewGuid() };
 
             var didCall = false;
-            Action<CallbackForTest> action = delegate(CallbackForTest cb)
+            void action( CallbackForTest cb )
```

Comment:
this change seem pointless - converting the delegate to a local function, only to have to cast it back to the delegate a couple lines down